### PR TITLE
use goreleaser for packaging .deb release

### PR DESCRIPTION
**Pull request by cryptix**
updates #203

TODO:
- [x] builds a `.deb` that can be installed
- [x] embeds docs/README and example config for nginx
- [x] creates systemd service that can be enable easily
- [x] post-install help text
- [x] #52 
- [x] Archlinux AUR for the totally wired: https://aur.archlinux.org/packages/go-ssb-room-git/
- [x] fixed embedded build information (version and commit)
- [x] check licenses of all the deps ( https://github.com/ssb-ngi-pointer/go-ssb-room/pull/222 )
- [x] maintainer email
- [x] rethink README.md and belonging assets

### CODE REVIEW

**Comment by cryptix** [comment]
Things to review in particular: changes to the readme and docs.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Copyright 2021 SSB NGI-Pointer Tream <go-ssb-room@ssb-ngi-pointer.email>
```
Comment (staltz) [inline LICENSE:1]:
```suggestion
-Copyright 2021 SSB NGI-Pointer Tream <go-ssb-room@ssb-ngi-pointer.email>
 Copyright 2021 SSB NGI-Pointer Team <go-ssb-room@ssb-ngi-pointer.email>
```

Comment (staltz) [reply LICENSE:1]: Tream => Team

File: docs/debian/postinstall.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# create a user to run the server as
+adduser --system --home /var/lib/go-ssb-room go-ssb-room
+chown go-ssb-room /var/lib/go-ssb-room
+
+# welcome message
+cat <<EOF
+> Welcome !
+
+go-ssb-room has been installed as a systemd service.
```
Comment (staltz) [inline docs/debian/postinstall.sh:9]: Nice

**Review by staltz — APPROVED** [review]
All good!

**Pull request merged**